tests/components/vive-focus-controls.test.js

```diff
@@ -30,7 +30,7 @@ suite('vive-focus-controls', function () {
       var addEventListenersSpy = sinon.spy(component, 'addEventListeners');
       var injectTrackedControlsSpy = sinon.spy(component, 'injectTrackedControls');
 
-      el.sceneEl.systems['tracked-controls'].controllers = [];
```

Comment:
Would be nice if tracked-controls adopted all the properties/methods from `tracked-controls-whatever`. Would maintain backwards compatibility. I have a lot of components that read `axis` and would be nice if they didn't have to check the existence of either component. Both webvr and webxr have the `axis` property

Reply:
there's no axis and won't likely be one in the WebXR API. Components will have to target the minimum common denominator (triggerdown event) or do component / API detection. Would a flag `isWebVR` / `isWebXR` in `tracked-controls` help or better just one global boolean like `isWebXRAvailable`?

Reply:
For your components you will have to do feature detection already even with the `controller` reference in `tracked-controls`. WebVR gives you a Gamepad and WebXR an InputSource

Reply:
For your components probably the easiest is to do A-Frame version detection and access `tracked-controls` for A-Frame < 0.9.0 and `tracked-controls-webvr` >= 0.9.0. The rest of your code should remain the same.

Reply:
An alternative is to start making a more abstract API to retrieve necessary info without accessing internal variables. A bit worry to over do it prematurely.